pyproject.toml

```diff
@@ -21,15 +21,13 @@ pandas = "^1.5.1"
 pytest-xdist = { version = "^3.0.2", extras = ["psutil"] }
 setproctitle = "^1.3.2"
 pysha3 = "^1.0.2"
-web3 = "^5.31.1"
+web3 = "6.0.0b2"
 python-dotenv = "^0.21.0"
-eth-utils = "<2.0.0"
-eth-abi = "<3.0.0"
 eth-account = "<0.8.0"
-eth-keys = "<0.4.0"
 autoflake = "^2.0.0"
 pycryptodome = "^3.16.0"
-rlp = "<3"
+rlp = "<=3"
+py-evm = "0.6.1a2"
```

Comment:
- starknet-py depends on typing-extensions (>=4.3.0,<5.0.0), which forces the use of py-evm = 0.6.1a2.
- py-evm = 0.6.1a2 depends on rlp (>=3,<4), eth-utils (>=2.0.0,<3.0.0) and eth-keys (>=0.4.0,<0.5.0).
- web3 = 5.31.3 depends on eth-utils (>=1.9.5,<2.0.0), so need to switch to web3 = 6.0.0b2 which depends on eth-abi (>=3.0.0,<4.0.0).

Reply:
why have you deleted the deps to `eth-utils`, `eth-abi` and `eth-keys`? even though they are as you said installed by `web3` we need to add them if we directly use them